generators/app/prompts.js

```diff
@@ -117,169 +117,26 @@ function askForOperations(generator) {
     const prompts = [
         {
             when: !generator.options['skip-prompts'],
-            type: 'list',
-            name: 'generationType',
-            message: 'Which type of generation do you want?',
+            type: 'confirm',
+            name: 'cleanup',
+            message: 'do you want to reset your kafka configuration?',
             choices: generationTypeChoices(),
-            default: [constants.BIGBANG_MODE]
+            default: false
         }
     ];
 
     const done = generator.async();
     try {
         generator.prompt(prompts).then(props => {
-            generator.props.generationType = props.generationType;
-            if (!props.generationType || props.generationType === constants.BIGBANG_MODE) {
-                askForBigBangOperations(generator, done);
-            } else if (props.generationType === constants.INCREMENTAL_MODE) {
-                askForIncrementalOperations(generator, done);
-            }
+            generator.props.cleanup = props.cleanup;
+            askForIncrementalOperations(generator, done);
```

Comment:
```suggestion
            askForOperations(generator, done);
```

Reply:
i rename this method, to avoid the use of incremental.